src/types.d.ts

```diff
@@ -21,10 +21,6 @@ declare type JsRequestType = "ipc"
 
 // see https://bitwiseshiftleft.github.io/sjcl/doc/symbols/sjcl.bitArray.html
 // type that is used by sjcl for any encryption/decryption operation
-// TODO these should be exported by tutanota-crypto
-declare type BitArray = number[]
-declare type Aes128Key = BitArray
-declare type Aes256Key = BitArray
 declare type SignedBytes = number[]
 declare type Base32 = string
```

Comment:
```suggestion

```
These types are also from `tutanota-crypto`. It might be a good idea to remove them here and export them from  `tutanota-crypto` as well. `SignedBytes` is from `packages/tutanota-crypto/lib/hashes/Bcrypt.ts` and `Base32` from `packages/tutanota-crypto/lib/misc/TotpVerifier.ts`.

They are currently unused however so just deleting them might suffice too.